### PR TITLE
Add note that general usage statistics submission is disabled with FIPS

### PR DESCRIPTION
**Pull request by daniel-beck**
Amends #8483 with the change suggested in https://github.com/jenkinsci/jenkins/pull/8483#discussion_r1339673531.

Unsure about wording, @jtnord WDYT?

> Screenshot 2023-10-06 at 10 22 16

### Testing done

Opened the form.

### Proposed changelog entries

too minor

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Review by jtnord — APPROVED** [review]
I can not think of any better wording

**Review by jtnord — APPROVED** [review]
I can not think of any better wording

**Review by MarkEWaite — APPROVED** [review]
Thanks!

**Comment by MarkEWaite** [comment]
This PR is now ready for merge.  We will merge it after approximately 24 hours if there is no negative feedback.

/label ready-for-merge

File: core/src/main/resources/hudson/model/UsageStatistics/global.properties [hunk 1]
```diff
@@ -24,6 +24,8 @@ statsBlurb=\
   Help make Jenkins better by sending anonymous usage statistics and crash reports to the Jenkins project
 statsBlurbFIPS=\
   Help make Jenkins better by sending telemetry to the Jenkins project
+statsDescriptionFIPS=\
+  General usage statistics are not submitted when FIPS-140 compliance is requested.
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/UsageStatistics/global.properties:28]:
```suggestion
  General usage statistics are never submitted when FIPS-140 compliance is requested.
```

To clarify it's independent of this setting?

Comment (MarkEWaite) [reply core/src/main/resources/hudson/model/UsageStatistics/global.properties:28]: I'm OK with either word, though I didn't infer the independence of the property from the change of wording.  My reading treated those two words as synonyms in that context.

**Comment by MarkEWaite** [comment]
Proceeding with the merge as currently phrased.  We can rephrase in a later pull request if needed

**Pull request merged**